relint/__main__.py

```diff
@@ -42,6 +42,9 @@ def parse_args(args=None):
     parser.add_argument(
         "-W", "--fail-warnings", action="store_true", help="Fail for warnings."
     )
+    parser.add_argument(
+        "--ignore-warnings", action="store_true", help="Ignore warnings."
```

Comment:
The help text isn't helpful, since it's identical to the argument itself. Your PR description is a good start. Please avoid repeating words, since they will likely not add any new context.

Reply:
Totally makes sense, I updated the wording. 👍